docs/kola-guide.md

```diff
@@ -0,0 +1,65 @@
+# Kola Introduction Documentation
+
+This document goes through the basics of how kola tests work and how someone just beginning can get started.
+
+### What is Kola?
+
+Kola is a framework for testing software integration in CoreOS systems across multiple platforms. It is primarily designed to operate within the CoreOS Assembler for testing software that has landed in the OS image.
+
+> [Additional documentation about Kola](https://github.com/coreos/coreos-assembler/tree/main/docs/kola)
+
+### Steps to run a test from start to finish
+
+ 1. `cosa run` (Creates and brings you into the shell of a QEMU instance, running the provided version of FCOS or RHCOS)
+ 2. `cosa kola list` (Here we can see all of the tests that we can do)
+ 3. `cosa kola run <test-pattern>` (This will run a specific test and check to see if a test passes)
+ 4. `cosa kola run ext.config*` (This will run all the tests that start with `ext.config`)
+
+### More information on tests
+
+After you run the kola test, you can find more information in tmp/kola/<test-name> about the test that just ran, as the following file logs . They will help you to debug the problem and will certainly give you hints along the way
+
+1. `journal.txt`
+2. `console.txt`
+3. `ignition.json`
+4. `gunzip journal-raw.txt.gz`
+
+### Extended artifacts
+
+1. Extended artifacts need additional forms of testing (You can pass the ignition and the path to the artifact you want to test)
+2. `cosa kola run -h` (this allows you to see the commands yourself and what syntax is needed)
+3. `cosa buildextend-"name_of_artifact` (An example of building an artifact before testing it)
+4. `kola run -p <platform>` Is the most generic way of testing extended artifacts, this is mostly useful for the cloud platforms
+5. For running the likes of metal/metal4k artifacts there's not much difference than running `kola run` from the coreos-assembler
+6. `cd builds/latest/` (This will show your latest build information)
```

Comment:
I'd use `cosa list` here; it will show you the most recent CoreOS builds that have been made and the artifacts created:

```
[coreos-assembler]$ cosa list
35.20220302.dev.0
   Timestamp: 2022-03-02T14:26:11Z (5 days, 1:59:15 ago)
   Artifacts: ostree qemu
      Config: pr/origin/1553 (bedbe76a65b4) (dirty)

35.20220223.dev.0
   Timestamp: 2022-02-23T14:06:22Z (12 days, 2:19:04 ago)
   Artifacts: ostree qemu
      Config: testing-devel (7413ad404988)

35.20220202.dev.0
   Timestamp: 2022-02-02T18:50:39Z (32 days, 21:34:47 ago)
   Artifacts: ostree qemu
      Config: testing-devel (6d7b9ad5c61a)
```

Reply:
sounds good, so should I just copy the contents of the kola.md file and add what I've done on the kola testiso aswell as the debug artifacts and just create a new PR?

Reply:
You can reuse this PR if you like or create a new one...whatever is easiest for you.